DashWallet/Sources/UI/Setup/SecureWallet/VerifiedSuccessfully/DWVerifiedSuccessfullyViewController.m

```diff
@@ -63,7 +63,7 @@ - (BOOL)requiresNoNavigationBar {
 #pragma mark - Actions
 
 - (IBAction)continueButtonAction:(id)sender {
-    [self.delegate secureWalletRoutineDidVerify:self];
+    [self.delegate secureWalletRoutineDidFinish:self];
```

Comment:
We had a bug with this callback - if the user verifies the passphrase and then dismisses the verify screen manually instead of clicking "Continue", the shortcuts on the home screen won't update.

This affects CrowdNode too, so I figured it's better to split this callback into "verified" and "finished".